ts/parts/Tooltip.ts

```diff
@@ -345,6 +346,13 @@ H.Tooltip.prototype = {
          */
         this.isHidden = true;
 
+        /**
+         * @readonly
+         * @name Highcharts.Tooltip#isSticky
+         * @type {boolean}
+         */
+        this.isSticky = (options.sticky || false);
```

Comment:
We don't really need a property on the `Tooltip` since this is a simple reflection of the option. Instead, we can add the default value `false` directly to the tooltip default options, and read from `options.sticky` whenever needed. That is the common and simplest pattern as long as we're not dealing with options that depend on some condition.

Reply:
This should be solved.